### DISCUSSION TITLE
[Q&A] Docker 安装，配置好环境变量后，为何返回值是空白？

### DISCUSSION THREAD

**Discussion by iAmP1us** [lobehub/lobehub#623] [Self Hosting] [answered]
### 💻 系统环境 | Operating System

Other Linux

### 🌐 浏览器 | Browser

Chrome

### 🐛 问题描述 | Bug Description

![QQ截图20231212105355](https://github.com/lobehub/lobe-chat/assets/86154051/970401f1-a1e8-490a-af54-05113705499b)
如上图，环境变量的API代理地址和key都配置好了，但是无论3.5还是4，返回值都是空白。
不知道是不是bug。

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
你的 api url 是否没加 `v1`

[a comment by an automated account is not shown]

**Comment by iAmP1us** [comment]
> 你的 api url 是否没加 `v1`

我用的第三方API地址，没有v1，但是我用隔壁chatgpt next web同样的配置，是正常的。

[a comment by an automated account is not shown]

**Comment by arvinxx** [comment]
可以私戳我，我来测试下

[a comment by an automated account is not shown]

**Comment by LBEILC** [comment]
> > 你的 api url 是否没加 `v1`
> 
> 我用的第三方API地址，没有v1，但是我用隔壁chatgpt next web同样的配置，是正常的。

隔壁确实是不需要加v1的,但是这边需要加上,完整的接口代理地址配置就是 地址/v1

[a comment by an automated account is not shown]

**Comment by iAmP1us** [comment, marked as answer]
> > > 你的 api url 是否没加 `v1`
> > 
> > 
> > 我用的第三方API地址，没有v1，但是我用隔壁chatgpt next web同样的配置，是正常的。
> 
> 隔壁确实是不需要加v1的,但是这边需要加上,完整的接口代理地址配置就是 地址/v1

解决了，确实是v1的问题，OPENAI_PROXY_URL=https://api-proxy.com/v1 ，我看到这条的时候第一反应，没考虑到v1，因为openai本来自带了一个v1，谢谢各位。

[a comment by an automated account is not shown]

**Comment by kokojacket** [comment]
@arvinxx 在使用gpt-4-1106-preview时仍然返回空白，结尾已经添加了V1，其他模型无异常
![image](https://github.com/lobehub/lobe-chat/assets/74887370/441d4c50-ccc0-432a-ac6e-7a60f934f8ab)

**Reply by arvinxx** [reply]
是哪家的api？

**Reply by kokojacket** [reply]
填写的one api地址，oneapi中部署了两家。https://api.fast-tunnel.one/，以及https://github.com/chatanywhere/GPT_API_free这个项目的免费和收费api

**Comment by chudaozhe** [comment]


**Reply by arvinxx** [reply]
确定讯飞星火的接口配置正确么？

**Comment by rz467fzs7d** [comment]
手动配置就正确，配置到docker环境变量就和问题描述一样
docker环境变量中带不带/v1都试过了 

请问日志从哪里啦？我看docker输出的日志只有nodejs服务的常规启动日志，没有应用日志
